libs/core/src/lib/multi-combobox/multi-combobox.component.ts

```diff
@@ -359,6 +379,11 @@ export class MultiComboboxComponent<T = any> extends BaseMultiCombobox<T> implem
         this._openDataStream();
     }
 
+    /** @hidden */
+    ngOnDestroy(): void {
+        this._langSubscription.unsubscribe();
+    }
+
```

Comment:
maybe you remove _langSubscription, directly subscribe _lang$ and use takeUntilDestory. with that way you remove langSub variable and onDestory hook.

Reply:
@mikerodonnell89 everything looks good, i just left only one comment related on onDestory

Reply:
Unfortunately for this PR, takeUntilDestroyed was introduced in Angular 16